.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 
-max-line-length = 88
\ No newline at end of file
+max-line-length = 120
```

Comment:
We should restrain from changing this setting since this is a common setting across our team. Is there a particular reason for why do we need to change this?